Makefile

```diff
@@ -15,9 +15,9 @@ help:
 	@echo "  DEBUG          -- debug flag, if any ($(DEBUG))"
 
 .PHONY: test
-test: e2e-test
+test: e2e-local
 
-.PHONY: e2e-test
+.PHONY: e2e-local
```

Comment:
Do you want to squash the commits together, so we don't merge the fixups too?

Reply:
done